packages/appstore/src/components/modals/open-positions-svg-modal/open-positions-svg-modal.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+
+type TOpenPositionsSVGModal = {
+    market_type: string;
+    open_order_position_status: boolean;
+    is_modal_open: boolean;
+    setModalOpen: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const OpenPositionsSVGModal = ({
+    market_type,
+    open_order_position_status,
+    is_modal_open,
+    setModalOpen,
+}: TOpenPositionsSVGModal) => {
+    const onClick = () => {
+        setModalOpen(false);
+    };
+    const title = open_order_position_status ? 'No new positions' : 'Account closed';
+    const content = open_order_position_status
+        ? 'You can no longer open new positions with your MT5 {{account_type}} SVG account. Please use your MT5 {{account_type}} BVI account to open new positions.'
```

Comment:
shouldn't this "bvi" dynamic?
it can be BVI or Vanuatu. isnt

Reply:
Thank you for pointing this out 😊